public/js/core.js

```diff
@@ -59,8 +59,13 @@ tarchive.controller('CoreController', ['$scope', '$filter', 'API', 'NgTableParam
 
 	$scope.askForKey = function() {
 		swal({
-			title: "Enter your key!",
-			text: "Enter the key provided by the bot:",
+			html: true,
+			title:	'Welcome to Tarchivepanel!',
+			text:		'What is Tarchivebot? <br><br> A Telegram bot that archives your chat and provides you with a web interface to look up messages and statistics. <br><br> Add the bot using this link: ' +
+							'<a href="https://t.me/Tarchivebot" target="_blank">Tarchivebot</a><br>' +
```

Comment:
Ah yes, I totally forgot about this. Ideally certain parts of the welcome dialog should indeed be read out of the [config file](https://github.com/PROGRADE-Tech/Tarchivebot/blob/master/config/config.json.example#L4).

This would require passing the contents over to Express, Pug, and lastly the core.js script. However I'd be totally fine with having that solved in another PR, thoughts @Hoi15A ?